src/OrchardCore.Modules/OrchardCore.Contents/Views/AdminTabs.cshtml

```diff
@@ -9,7 +9,7 @@
     <ul class="nav nav-tabs flex-column flex-md-row" role="tablist">
         @foreach (var tab in tabs)
         {
-            var tabId = $"tab-{tab}-{Model.ContentItem.ContentItemId}".HtmlClassify();
+            var tabId = $"tab-{tab}-{Model.ContentItem?.ContentItemId}".HtmlClassify();
```

Comment:
An id here is not required, it is just useful if you want to target these with javascript.

And for user shapes, there isn't an easy id, but we don't need it.